cosmo_tester/framework/util.py

```diff
@@ -39,6 +40,24 @@
 CLOUDIFY_AUTH_TOKEN_HEADER = 'Authentication-Token'
 
 
+def get_file_name_from_url(url):
+    return url.split('/')[-1]
+
+
+def download_file(url, destination=''):
```

Comment:
it is somewhat surprising for  a general purpose download method to cache results, please remove that if exists part. also, please use requests instead of urllib